python/pyspark/sql/connect/plan.py

```diff
@@ -1049,21 +1049,23 @@ def __init__(self, query: str, args: Optional[Union[Dict[str, Any], List]] = Non
         self._query = query
         self._args = args
 
+    def __to_expr(self, session: "SparkConnectClient", v: Any) -> proto.Expression:
```

Comment:
I think we'd better rename it `_to_expr`.

[the python standard](https://peps.python.org/pep-0008/#method-names-and-instance-variables) say:


> Note: there is some controversy about the use of __names


> In addition, the following special forms using leading or trailing underscores are recognized (these can generally be combined with any case convention):

> `_single_leading_underscore`: weak “internal use” indicator. E.g. from M import * does not import objects whose names start with an underscore.
> `single_trailing_underscore_`: used by convention to avoid conflicts with Python keyword, e.g.
tkinter.Toplevel(master, class_='ClassName')
> `__double_leading_underscore`: when naming a class attribute, invokes name mangling (inside class FooBar, __boo becomes _FooBar__boo; see below).
> `__double_leading_and_trailing_underscore__`: “magic” objects or attributes that live in user-controlled namespaces. E.g. __init__, __import__ or __file__. Never invent such names; only use them as documented.